src/libcore/iter/traits/iterator.rs

```diff
@@ -1,10 +1,16 @@
+// ignore-tidy-filelength
+// This file almost exclusively consists of the definition of `Iterator`. We
+// can't split that into multiple files.
```

Comment:
Well, technically we can split definition of Iterator into multiple files and `include!` them into source...

Reply:
But we won't do that, right?

Reply:
Well, that's up to libs team to decide

Reply:
I wouldn't want to do that. That seems worse.